source/scss/components/_backdrop.scss

```diff
@@ -0,0 +1,11 @@
+.backdrop {
+  background: #000000;
```

Comment:
Do we need both backgrounds in this class?

Reply:
Also, maybe we shoud use `background-color` instead of `background` here.

Reply:
According to the syntax definition, if background takes a color, it will change the background-color and keeps everything else on default. Is it for readability reasons, why you want to rather use background-color?

Reply:
The only reason is to have a standard way to do that, actually. On the same file, we're using both ways to change the background colour.